source/adapters/level_zero/command_buffer.cpp

```diff
@@ -616,13 +624,79 @@ bool canBeInOrder(ur_context_handle_t Context,
              ? (CommandBufferDesc ? CommandBufferDesc->isInOrder : false)
              : false;
 }
-} // namespace
 
-UR_APIEXPORT ur_result_t UR_APICALL
-urCommandBufferCreateExp(ur_context_handle_t Context, ur_device_handle_t Device,
-                         const ur_exp_command_buffer_desc_t *CommandBufferDesc,
-                         ur_exp_command_buffer_handle_t *CommandBuffer) {
+ur_result_t
+commandBufferCreateNew(ur_context_handle_t Context, ur_device_handle_t Device,
```

Comment:
After the implementation is finished, will try to figure out if the "New" and "Old" entrypoints should be merged or not. For now keeping the implementations separate to make it less confusing